packages/spotlib/spotlib.4.3.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Useful functions for OCaml programming used by @camlspotter"
+description: """\
+Spotlib is a small library package used for several softwares by Jun Furuse.
+It is almost a poor replication of Jane Street Core, but it is small."""
+maintainer: "jun.furuse@gmail.com"
+authors: "Jun Furuse"
+license: "MIT"
+homepage: "https://gitlab.com/camlspotter/spotlib"
+bug-reports: "https://gitlab.com/camlspotter/spotlib/-/issues"
+depends: [
+  "dune" {build & >= "2.0"}
+  "ocaml" {>= "4.12.0" & < "5.0.0"}
+  "ppx_test" {>= "1.8.0"}
+]
```

Comment:
According to your code (https://gitlab.com/camlspotter/spotlib/-/blob/8471ac15ea94f18470d5c7e1407221a81ceecb6d/xcurl/dune) this is required
```suggestion
]
depopts: [
  "curl"
]
```

Reply:
Fix in previous versions as well in https://github.com/ocaml/opam-repository/pull/22420